wordpress-mobile/WordPress-iOS
# Zendesk Mobile: code cleanup and comments

Ref #9020

This is just cleaning up some code and adding comments. No functional changes.

To test:
Just some sanity checks:
- Go to the Support view.
- Select 'WordPress Help Center' and verify it appears.
- Select 'Contact Us' and submit a ticket (I'll take care of it on the Zendesk side).
- Select 'My Tickets' and verify your ticket is there.

@etoledom - quite possibly the easiest PR you'll ever get from me! Would you mind? Thank you!!
